public_html/wp-content/mu-plugins/blocks/source/blocks.js

```diff
@@ -10,6 +10,10 @@ import './_z-index.scss'; // Have z-index values, similar to https://github.com/
 import './styles.scss'; // Common styles for WordCamp Blocks.
 import { BLOCKS } from './blocks/'; // Trailing slash required to differentiate the folder from the file.
 
-BLOCKS.forEach( ( { NAME, SETTINGS } ) => {
+const enabledBlocks = BLOCKS.filter( ( b ) =>
```

Comment:
```suggestion
const enabledBlocks = BLOCKS.filter( ( block ) =>
```

`SHOULD` - Despite it's proximity to `BLOCKS`, I still don't think `b` is meaningful/descriptive enough.

Same for the line below.

Reply:
+1